blocks/condition/edit.tsx

```diff
@@ -48,90 +59,22 @@ export default function Edit({
       <InspectorControls>
         {/* @ts-ignore-next-line */}
         <PanelBody
-          title={__('Condition', 'wp-conditional-blocks')}
+          title={__('Conditions', 'wp-conditional-blocks')}
           initialOpen
         >
           {/* @ts-ignore-next-line */}
           <PanelRow>
             {/* @ts-ignore-next-line */}
-            <TextControl
-              label={__('Query', 'wp-conditional-blocks')}
-              help={__('Query condition, ie "is_home" or "is_category"', 'wp-conditional-blocks')}
-              onChange={(next) => setAttributes({ query: next })}
-              value={query}
-            />
-          </PanelRow>
-          {/* @ts-ignore-next-line */}
-          <PanelRow>
-            {/* @ts-ignore-next-line */}
-            <TextControl
-              label={__('Post', 'wp-conditional-blocks')}
-              help={__('Post condition, ie "is_content"', 'wp-conditional-blocks')}
-              onChange={(next) => setAttributes({ post: next })}
-              value={post}
-            />
-          </PanelRow>
-
-          {/* @ts-ignore-next-line */}
-          <PanelRow>
-            {/* @ts-ignore-next-line */}
-            <TextControl
-              label={__('Custom', 'wp-conditional-blocks')}
-              help={__('Custom condition, ie "is_column"', 'wp-conditional-blocks')}
-              onChange={(next) => setAttributes({ custom: next })}
-              value={custom}
-            />
-          </PanelRow>
-
-          {/* @ts-ignore-next-line */}
-          <PanelRow>
-            {/* @ts-ignore-next-line */}
-            <TextControl
-              label={__('Condition', 'wp-conditional-blocks')}
-              help={__('Any other condition', 'wp-conditional-blocks')}
+            <SelectControl
+              label={__('Conditionals', 'wp-conditional-blocks')}
```

Comment:
Double checking that you want to use "Conditional" here and "Condition" in the label. Should both be consistent?